roles/sssd_ad/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# defaults file for system_ldap
+ad_domain: "pu.win.princeton.edu"
+ad_test_user: "{{ omit }}"
```

Comment:
Supply a username, then default to omit for testing. `"{{ my_user | default(omit) }}"`